shared/src/commonMain/kotlin/co/nimblehq/avishek/phong/kmmic/data/remote/datasource/TokenRemoteDataSource.kt

```diff
@@ -2,20 +2,24 @@ package co.nimblehq.avishek.phong.kmmic.data.remote.datasource
 
 import co.nimblehq.avishek.phong.kmmic.data.remote.ApiClient
 import co.nimblehq.avishek.phong.kmmic.data.remote.body.RefreshTokenRequestBody
+import co.nimblehq.avishek.phong.kmmic.data.remote.body.LogInRequestBody
 import co.nimblehq.avishek.phong.kmmic.data.remote.model.TokenApiModel
-import co.nimblehq.avishek.phong.kmmic.data.remote.path
-import io.ktor.client.request.*
 import io.ktor.http.*
 import kotlinx.coroutines.flow.Flow
 
 interface TokenRemoteDataSource {
 
     fun refreshToken(body: RefreshTokenRequestBody): Flow<TokenApiModel>
+    fun logIn(body: LogInRequestBody): Flow<TokenApiModel>
 }
 
 class TokenRemoteDataSourceImpl(private val apiClient: ApiClient) : TokenRemoteDataSource {
 
     override fun refreshToken(body: RefreshTokenRequestBody): Flow<TokenApiModel> {
-        return apiClient.responseBody("/v1/oauth/token", HttpMethod.Post, body)
+        return apiClient.responseBody("api/v1/oauth/token", HttpMethod.Post, body)
+    }
+
+    override fun logIn(body: LogInRequestBody): Flow<TokenApiModel> {
+        return apiClient.responseBody("api/v1/oauth/token", HttpMethod.Post, body)
```

Comment:
Please define the API path

Reply:
@suho, did you mean a constant for the API path?

Reply:
@phongvhd93 Yes, currently, you are using the same endpoint string for 2 APIs